Makefile

```diff
@@ -287,6 +299,8 @@ docker run -dt --privileged \
 	--add-host "$(REGISTRY_HOST):$(shell $(IP_CMD) $(MASTER_CTR)1 2>/dev/null || echo 127.0.0.1)" \
 	$(DOCKER_IMAGE);
 sleep 2;
+docker exec $(1)$(2) mkdir -p /var/lib/dcos
```

Comment:
Summary: This is one of the common files in https://github.com/dcos/dcos/blob/master/packages/mesos/extra/dcos-mesos-slave.service. We disable the systemd of Mesos if we are not using systemd on the host because that requires systemd on the host.

We use a "common" file so that we only write to one file for agent and public agent nodes.